package/rasplex/rasplex.mk

```diff
@@ -103,8 +120,21 @@ ifdef RASPLEX_SKIN_NAME
 		$(RASPLEX_SKIN_SITE)/$(RASPLEX_SKIN_NAME)/$(RASPLEX_SKIN_FILE)
 endif
 
+# Skin configuration for Plex Black Edition
+ifdef BR2_PACKAGE_RASPLEX_SKIN_PLEX_BLACK_EDITION
+	RASPLEX_SKIN_NAME = skin.plex_black_editionHT
+	RASPLEX_SKIN_VERSION = v16.11.24
+	# URL information stored in /storage/.plexht/userdata/Database/Addons15.db
+	RASPLEX_SKIN_SITE = https://addons.openpht.tv/openpht-1.6
+	RASPLEX_SKIN_FILE = $(RASPLEX_SKIN_NAME)-$(RASPLEX_SKIN_VERSION).zip
+	ZIP_NAME = HQlrwa
+	# # Download skin if required by configuration
+	RASPLEX_EXTRA_DOWNLOADS = https://goo.gl/HQlrwa
+endif
+
```

Comment:
Old code, now we are downloading the skin from the official source https://addons.openpht.tv/openpht-1.8, not from https://goo.gl anymore.
I guess it is a merge conflict error.